lib/pbench/client/__init__.py

```diff
@@ -321,31 +328,27 @@ def connect(self, headers: Optional[dict[str, str]] = None) -> None:
         self.endpoints = response.json()
         assert self.endpoints
 
-    def login(self, user: str, password: str) -> JSONMap:
-        """Login to a specified username with the password, and store the
-        resulting authentication token.
+        # Create an OIDCAdmin object and confirm the connection was successful
+        self.oidc_admin = OIDCAdmin(server_url=self.endpoints["openid"]["server"])
+        assert self.oidc_admin.get_admin_token()["access_token"]
```

Comment:
I recommend against using `assert` in this of code.  (While I'm sure that this will be used by functional tests, I wouldn't rush to consider this code itself to be test code.)

Ditto line 329, as far as that goes....

We seem to be comfortable reporting errors by exception at line 327, so just invoke `raise` with something appropriate here if the token isn't available.

Ditto line 348.